plugins/woocommerce/tests/php/includes/rest-api/Controllers/Version3/class-wc-rest-paypal-standard-controller-test.php

```diff
@@ -22,6 +22,15 @@ public function setUp(): void {
 		wp_set_current_user( $this->user );
 	}
 
+	/**
+	 * Tear down the test environment.
+	 */
+	public function tearDown(): void {
+		parent::tearDown();
+
+		wp_set_current_user( 0 );
```

Comment:
Nit: I think we should clean up the local state before we call the parent cleanup.
```suggestion
		wp_set_current_user( 0 );

		parent::tearDown();
```